examples/06_scope_functions/04_apply.md

```diff
@@ -0,0 +1,32 @@
+# apply
+
+`apply` executes a given block on an object and returns the object itself. Inside the block, the object is referenced by `this`.
```

Comment:
`apply` executes a block on an object and returns the object itself. Inside the block, the object is referenced by `this`.